package.json

```diff
@@ -67,7 +67,7 @@
     "@types/yargs": "^15.0.5",
     "assert-rejects": "^1.0.0",
     "c8": "^7.2.1",
-    "codecov": "^3.7.0",
+    "codecov": "^3.7.1",
```

Comment:
I think we've actually moved away from requiring this, in favor of the `codecov` GitHub Action.  Should we drop it?   Also noticed `assert-rejects` up there, we don't need that anymore either :P

Reply:
Yeah, let me drop both (in a separate PR).